WooCommerce/src/test/kotlin/com/woocommerce/android/ui/themes/ThemePickerViewModelTest.kt

```diff
@@ -207,4 +210,19 @@ class ThemePickerViewModelTest : BaseUnitTest() {
 
         verify(crashLogger).sendReport(any(), anyOrNull(), anyOrNull())
     }
+
+    @Test
+    fun `given current theme is loaded, when showing themes, then remove current theme from the list of items`() =
+        testBlocking {
+            setup(isFromStoreCreation = false) {
+                whenever(themeRepository.fetchThemes())
+                    .thenReturn(Result.success(listOf(sampleTheme, sampleTheme2)))
+                whenever(themeRepository.fetchCurrentTheme()).thenReturn(Result.success(currentTheme))
+            }
+
+            val viewState = viewModel.viewState.runAndCaptureValues { advanceUntilIdle() }.last()
+            val carouseItems = (viewState.carouselState as ThemePickerViewModel.CarouselState.Success).carouselItems
+
+            assertThat((carouseItems.first() as CarouselItem.Theme).themeId).isNotEqualTo(currentTheme.id)
```

Comment:
np, a more readable check would be:
```suggestion
            assertThat((carouseItems)).noneMatch { it is CarouselItem.Theme && it.themeId == currentTheme.id }
```

This will be also more explicit, as the current variant tests only against the first item, and for example if the logic of the ViewModel was to change the order of the items, we'll still have a green test even if `currentTheme` was still included in the list just not on the first position.